pom.xml

```diff
@@ -505,16 +513,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
As far as I can tell, the version of the parent pom in use here has switched over to SpotBugs, so we can delete this configuration completely.